alibi_detect/cd/pytorch/lsdd_online.py

```diff
@@ -1,8 +1,10 @@
+from webbrowser import get
```

Comment:
Accidental addition?

Reply:
Yes, vscode is playing tricks with me. Let me remove it.

Reply:
Ah good old vscode!